scripts/python/archive-repositories.py

```diff
@@ -0,0 +1,35 @@
+import os
+from datetime import datetime
+from lib.MojGithub import MojGithub
+
+# This file assigns archives all repositories which have had no commits from a certain datetime
+# The goal is clean the ministryofjustice GitHub organization.
+
+# TODO:
+# Currently only prints repos, it should archive them
+# Whitelist is needed before enabling above ^
+
+## Config
+# Change this to point at a different GitHub organization
+organization        = "ministryofjustice"
+org_token           = os.getenv('ADMIN_GITHUB_TOKEN')
+
+# The date in which repositories should be archived before
+archive_date        = "2018-01-17"
```

Comment:
Should this be relative to todays date rather than hardcoded, e.g. using `dateutil.relativedelta` to get four years ago?

Something like this potentially:
```python
archived_date = datetime.now() - relativedelta(years=4)
```

Reply:
I have used this to minus a datetime and turn it to iso format:
from_date = (datetime.now() - timedelta(weeks=(9 * 4))).isoformat()

Reply:
Great idea, cheers - will implement now.

Reply:
https://github.com/ministryofjustice/operations-engineering/pull/470/commits/d2e3de37ebb9226e1dd22926a7d596b5fe9fec9c